.clang-tidy

```diff
@@ -6,6 +6,7 @@ Checks: >
   performance-*,
   portability-*,
   readability-*,
+  -modernize-use-trailing-return-type
```

Comment:
With this we remove that annoying trailing return changes on main (or future methods/functions)

Reply:
Amazing!